telco-hub/configuration/README.md

```diff
@@ -0,0 +1,218 @@
+
+# Automated installation
+The full telco hub configuration can be applied using an ArgoCD application pointing to the kustomization.yaml in this directory.
+
+## Pre-requisites
+* An OpenShift cluster with the gitops-operator (ArgoCD) installed
+  * Note that the reference configuration includes a ClusterRole for ArgoCD which grants the necessary permissions for installing the remainder of the reference. This updates the currently running      ArgoCD application to allow it to complete the full synchronization.
+* If ODF will be used in "internal" mode, nodes with available storage for ODF must be labeled
+  `cluster.ocs.openshift.io/openshift-storage=`
+* All files/directories in this tree are available in a git repository along with any necessary kustomize overlay for your environment.
+* Configured and existing Openshift CatalogSources for `redhat-operators-disconnected` and `certified-operators-disconnected`.
+
+## Init phase (prepare ArgoCD)
+
+ArgoCD is one of the main key components of the Telco Hub. At the same time, we can deploy the Telco Hub using ArgoCD (recommended procedure). Therefore, to have a Telco Hub with ArgoCD, first, we have to have ArgoCD to create the Telco Hub. This is the init phase.
+
+In this init phase we will install ArgoCD with the existing `reference-crs` for gitops. But, you could provide ArgoCD differently, or maybe, you already have ArgoCD in your cluster. 
+
+In case you want to proceed with the existing `reference-crs` for gitops (recommended): 
+
+```bash
+oc apply -f reference-crs/required/gitops/clusterrole.yaml \
+  -f reference-crs/required/gitops/clusterrolebinding.yaml \
+  -f reference-crs/required/gitops/gitopsNS.yaml \
+  -f reference-crs/required/gitops/gitopsOperatorGroup.yaml \
+  -f reference-crs/required/gitops/gitopsSubscription.yaml
+```
+
+Wait the operator to be installed:
```

Comment:
If you use the "kustomize build ...." above it creates the application and starts sync'ing the entire configuration. So this "wait" step is already built into the waves that are in the CRs. No further steps are needed.

Reply:
Oka, now I made ArgoCD fully self-managed by ArgoCD

Reply:
Does that mean we can update the oc apply -f of all the resources with the simpler (and more maintainable) ?
oc apply -k example-overlays-config/gitops/kustomization.yaml

Reply:
it is there but the ArgoCD application phase. For the init phase, where we dont even have Openshift Gitops installed, we need to apply only the Manifests to install Openshift GitOps.